### PR TITLE
Fix starting the game from the lobby in the editor

### PR DESCRIPTION
**Pull request by NoooneyDude**
- Fixes being unable to start the game from the lobby in the editor.

> Tested with a client on headless.

### CODE REVIEW

**Comment by corp-0** [comment]
IMG_20220430_055642
Look who is back, it just suddenly decided to work again

**Comment by corp-0** [comment]
It was asking for braces on a return inside an if. We have agreed on allowing that when it is a oneliner so I disabled the pattern. Push an empty commit to retrigger the analysis so you get that nice success green

File: UnityProject/Assets/Scripts/Core/Editor/SceneAutoLoader.cs [hunk 1]
```diff
@@ -1,100 +1,111 @@
...
+			// Save current scene to return to it after play mode.
+			PreviousSceneName = UnityEngine.SceneManagement.SceneManager.GetActiveScene().name;
+			PreviousScenePath = UnityEngine.SceneManagement.SceneManager.GetActiveScene().path;
 
-			// User pressed play -- autoload online scene.
-			PreviousScene = EditorSceneManager.GetActiveScene().path;
-			if (EditorSceneManager.SaveCurrentModifiedScenesIfUserWantsTo())
+			// if the scene is startup or lobby, load lobby scene
+			var initialScene = UnityEngine.SceneManagement.SceneManager.GetActiveScene().path;
+			if (initialScene.Contains("StartUp") || initialScene.Contains("Lobby"))
```
Comment (Bod9001) [inline UnityProject/Assets/Scripts/Core/Editor/SceneAutoLoader.cs:80]: what about online scene?

Comment (NoooneyDude) [reply UnityProject/Assets/Scripts/Core/Editor/SceneAutoLoader.cs:80]: If it's not lobby or startup, it'll load `MasterScene` which is the online scene

Comment (Bod9001) [reply UnityProject/Assets/Scripts/Core/Editor/SceneAutoLoader.cs:80]: ok cool

**Review by corp-0 — APPROVED** [review]

**Pull request merged**